src/shared/components/Orgs/OrgItem.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Button } from 'antd';
+import { AccessControl } from '@bbp/react-nexus';
+import ListItem from '../Animations/ListItem';
+
+export interface OrgItemProps {
+  label: string;
+  description?: string;
+  numProjects?: number;
```

Comment:
I don't like this props name... Is this the number of projects? I would call it `numberOfProjects`